test/scripts/e2e_subs/e2e-app-extra-pages.sh

```diff
@@ -97,8 +94,13 @@ fi
 
 RES=$(${gcmd} app info --app-id ${APPID}  2>&1 || true)
 PROGHASH="Approval hash:         7356635AKR4FJOOKXXBWNN6HDJ5U3O2YWAOSK6NZBPMOGIQSWCL2N74VT4"
+EXTRAPAGES="Extra program pages:   1"
 if [[ $RES != *"${PROGHASH}"* ]]; then
-    date '+app-extra-pages-test FAIL the application info should succeed %Y%m%d_%H%M%S'
+    date '+app-extra-pages-test FAIL the application approval program hash is incorrect %Y%m%d_%H%M%S'
```

Comment:
Might be nice to migrate toward doing this at the top of e2e .sh programs
```
filename=$(basename "$0")
scriptname="${filename%.*}"
date "+${scriptname} start %Y%m%d_%H%M%S"
```
So you can use $scriptname here (and elsewhere). No big deal, just makes copying testing for modification easier.